### ISSUE TITLE
invalid sumo environment when applying causes error with no message

### ISSUE THREAD

**Issue by marionettist** [SumoLogic/sumologic-terraform-provider#14]
When valid Sumo credentials are set (access id and access key) but the sumo environment is not set and the default environment is not the right one, e.g. when the environment should have been set to 'eu' but the default of 'us2' is used, a terraform apply fails with a blank error message.

```
Error: Error applying plan:
1 error(s) occurred:
* sumologic_collector.test: 1 error(s) occurred:
* sumologic_collector.test: 
Terraform does not automatically rollback in the face of errors.
```

Will submit a PR

**Comment by frankreno** [comment]
Will close when we cut a new release.

**Comment by frankreno** [comment]
fixed in v0.2.0
